src/components/tag-view/index.js

```diff
@@ -15,13 +30,39 @@ export default class TagView extends Component {
               className='tagExitButton'
             />
           </Link>
```

Comment:
can you remove the Link tag as you already have the onClick using a redux action?